client/src/commands/removeWalletCommand.ts

```diff
@@ -45,6 +47,14 @@ export async function removeWallet(treeItem: WalletTreeItem): Promise<void> {
         await fs.remove(walletRegistryEntry.walletPath);
         await FabricWalletRegistry.instance().delete(walletRegistryEntry.name);
 
+        const gateways: FabricGatewayRegistryEntry[] =  FabricGatewayRegistry.instance().getAll();
```

Comment:
flag

Reply:
Just checked and `FabricGatewayRegistry.instance().getAll()` will always return an array (if called from the command, it's possible to be an empty array).